src/utils/sentry.ts

```diff
@@ -7,7 +7,7 @@ export const initSentry = () => {
       new Sentry.BrowserTracing({
         tracePropagationTargets: ['https://web3inbox-dev-hidden.vercel.app']
       }),
-      new Sentry.Replay(),
+      new Sentry.Replay()
```

Comment:
Side note: can we adjust this pls to only collect on errors (`replaysOnErrorSampleRate`) or remove if not necessary? 🙏 We keep blowing through the (small) allocation we have very quickly

Reply:
https://docs.sentry.io/platforms/javascript/session-replay/configuration/#general-integration-configuration

Reply:
It's not very useful, I never could make use of it while debugging. Removing